### PR TITLE
Add info how to setup module as a global module

### LINKED ISSUE #158: Wrong link to the latest version in documentation
On the readme page here https://github.com/sphereio/sphere-node-product-csv-sync/blob/master/README.md we have a link which leads to this address https://github.com/sphereio/sphere-node-product-csv-sync/archive/latest.zip

It should point to the latest version of this module but module on this address has version "0.6.7" instead.

### PR DESCRIPTION
**Pull request by junajan**
Fixes #158

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,26 +11,30 @@ Further you can change the publish state of products.
 
 # Setup
 
-* install [NodeJS](http://support.sphere.io/knowledgebase/articles/307722-install-nodejs-and-get-a-component-running) (platform for running application)
+* Install [NodeJS](http://support.sphere.io/knowledgebase/articles/307722-install-nodejs-and-get-a-component-running) (platform for running application).
```
Comment (Siilwyn) [inline README.md]: Let's remove the 'Setup' and 'From scratch' part completely.

File: README.md [hunk 2]
```diff
@@ -11,26 +11,30 @@ Further you can change the publish state of products.
...
 $ npm install
 ```
-* build javascript sources
+* Build javascript sources.
 ```bash
 $ grunt build
 ```
 
-### From ZIP
+### As a global module using NPM
```
Comment (Siilwyn) [inline README.md]: Rename to 'Install' or 'Setup' as you see fit.

File: README.md [hunk 3]
```diff
@@ -11,26 +11,30 @@ Further you can change the publish state of products.
...
 
-* Just download the ready to use application as [ZIP](https://github.com/sphereio/sphere-node-product-csv-sync/archive/latest.zip)
-* Extract the latest.zip with `unzip sphere-node-product-csv-sync-latest.zip`
-* Change into the directory `cd sphere-node-product-csv-sync-latest`
+* install [npm](http://gruntjs.com/getting-started) (NodeJS package manager, bundled with node)
```
Comment (Siilwyn) [inline README.md]: Only the install command would be enough.

Comment (junajan) [inline README.md]: Updated here 1bbe99b

File: README.md (hunk 2 above)
Comment (junajan) [inline README.md]: Updated here 1bbe99b

File: README.md (hunk 1 above)
Comment (junajan) [inline README.md]: Updated here 1bbe99b

**Comment by Siilwyn** [comment]
I've changed it a bit, you shouldn't use `sudo`: https://docs.npmjs.com/getting-started/fixing-npm-permissions

And as described in the issue we should also remove the 'zip' release thingy. @hisabimbola please take a final look.

**Pull request merged**